SU2_CFD/include/iteration/CIteration.hpp

```diff
@@ -101,6 +101,18 @@ class CIteration {
   void SetMesh_Deformation(CGeometry** geometry, CSolver** solver_container, CNumerics*** numerics_container,
                            CConfig* config_container, unsigned short kind_recording);
 
+   /*!
+    * \brief Run the mesh deformation algorithms.
+    * \author R. Sanchez
+    * \param[in] geometry - Geometrical definition of the problem.
+    * \param[in] solver_container - Container vector with all the solutions.
+    * \param[in] config - Definition of the particular problem.
+    * \param[in] iZone - Index of the zone.
+    * \param[in] kind_recording - Current kind of recording.
+    */
+  void SetMesh_StaticDeformation(CGeometry** geometry, CSolver** solver_container, CNumerics*** numerics_container,
+                                 CConfig* config_container, unsigned short kind_recording);
+
```

Comment:
Can you explain why you need a "static" deformation? Is it to freeze grid velocities during a dynamic simulation?

Reply:
First of all thank you very much for your comments! I will move the test cases in the appropriate location and I will try to reduce the size as much as possible. I can definitely try to prepare a tutorial, tomorrow (it is evening here) I will try to have a look at what is already present in order to follow the same style.

The static deformation is needed in the very beginning of an unsteady simulation if a non-zero initial deformation is present. As the simulation is unsteady the code would try to perform a dynamic mesh update, creating fictitious velocities as the initial velocity is set to zero everywhere. In this way I basically statically deform the mesh and then push back the solution at previous times to avoid this problem.

Reply:
I see that makes sense, since this is only needed on the first iteration would it be possible to achieve the same result by calling the normal mesh deformation routine and them zero-ing the grid velocities? We should avoid duplicating things as much as possible.